client/src/helpers/ehr-defs-grid.js

```diff
@@ -103,6 +101,27 @@ class EhrDefsWorker {
     return medPeriods
   }
 
+  getCaseStudyDataStatus (pageKey) {
+    const pd = this.getPageDefinition(pageKey)
+    const recHeader = pd.recHeader
+    return recHeader
+  }
+
```

Comment:
This method is no longer part of the code. It was removed in my previous PR.  Can you please remove from this PR and also check this file for other possible unintended changes.